.github/workflows/macos.yml

```diff
@@ -2,10 +2,17 @@ name: macos
 
 on:
   pull_request:
+    paths-ignore:
+      - "bundler/**"
+      - ".github/workflows/*bundler*.yml"
```

Comment:
Please also remove all these changes, I didn't suggest to skip any `rubygems` builds. Rubygems tests load the local `bundler` sometimes, and rubygems tests are fast anyways.